modules/api/src/main/java/io/fluo/api/config/FluoConfiguration.java

```diff
@@ -66,6 +66,7 @@
   // Administration
   private static final String ADMIN_PREFIX = FLUO_PREFIX + ".admin";
   public static final String ADMIN_ACCUMULO_TABLE_PROP = ADMIN_PREFIX + ".accumulo.table";
+  public static final String ADMIN_HDFS_ROOT_PROP = ADMIN_PREFIX + ".hdfs.root";
```

Comment:
Since this prop is only used for interpolation within the config file, do you think it needs a presence in this class?